spec/api-server.spec.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+describe("ApiServer", () => {
+    var apiServer, webServerMock;
+
+    beforeEach(() => {
+        var ApiServer, configMock;
+
+        ApiServer = require("../lib/api-server");
+        configMock = {
+            server: {
+                "baseUrl": "https://localhost:8080/",
+                "port": 8443
+            }
+        };
+        webServerMock = jasmine.createSpyObj("webServerMock", [
+            "configure",
+            "addRoute",
+            "startServer"
+        ]);
+        apiServer = new ApiServer(configMock, webServerMock);
+    });
+
+    it("starts a server", () => {
+        var next, req, res, routeCall;
+        res = jasmine.createSpyObj("resMock", [
+            "setHeader",
+            "send"
+        ]);
+        next = jasmine.createSpy("nextMock");
+        expect(webServerMock.configure).toHaveBeenCalledWith({
+            "baseUrl": "https://localhost:8080/",
+            "port": 8443
+        });
+        expect(apiServer).toEqual(jasmine.any(Object));
+        expect(webServerMock.addRoute).toHaveBeenCalledWith("get", "/", jasmine.any(Function));
+        routeCall = webServerMock.addRoute.mostRecentCall.args[2];
```

Comment:
The testing of the addition of the route should be in a separate test.  I would also recommend wrapping it in

    expect(() => {
        routeCall(req, res, next);
    }).not.toThrow();